### ISSUE TITLE
Slight Delay on Setting Wallpaper

### ISSUE THREAD

**Issue by mahyarmirrashed** [hyprwm/hyprpaper#213]
Upon launching, I see the default Hyprland wallpaper for about 0.5 seconds before Hyprpaper activates and sets my background. Is there an option for that where I can do it instantly? Or at least disable whatever Hyprland comes with by default?

## `hyprland.conf`

```
exec-once = /nix/store/...
exec-once=hyprpaper
```

## `hyprpaper.conf

```
preload=/nix/store/<hash>-space.png
wallpaper=,/nix/store<hash>-space.png
```

**Comment by mahyarmirrashed** [comment]
To fix this, you must add this in the `misc` section of your `hyprland.conf`:
```
disable_splash_rendering = true
disable_hyprland_logo = true
```
Ref: https://wiki.hyprland.org/Configuring/Variables/#misc

I made this ticket so that it can be added to the docs.

**Comment by vaxerski** [comment]
> Is there an option for that where I can do it instantly?

Yes, ask NASA to send you a supercomputer. Loading your image takes a moment.

**Comment by mahyarmirrashed** [comment]
@vaxerski I meant that I was still seeing the Hyprland wallpaper before the one I configured with Hyprpaper was loaded. I don't mind taking Hyprpaper to load mine, so long as I didn't see the default Hyprland one in between.

Your sassiness comes across as unhelpful and unwelcoming, which isn’t aligned with the collaborative spirit of open-source spaces.
